lib/wallet/test/unit/Cardano/Wallet/Shelley/TransactionSpec.hs

```diff
@@ -3030,15 +3021,17 @@ instance MonadRandom Gen where
     getRandomR range = mkGen (fst . randomR range)
     getRandomRs range = mkGen (randomRs range)
 
-data Wallet' = Wallet'
-    UTxOAssumptions
-    UTxO
-    AnyChangeAddressGenWithState
+data Wallet' =
+    Wallet' TxWitnessTag UTxOAssumptions UTxO AnyChangeAddressGenWithState
```

Comment:
Suggestion that retains the original diff-friendly format, and thus makes a more minimal change:
```patch
  data Wallet' = Wallet'
+     TxWitnessTag
      UTxOAssumptions
      UTxO
      AnyChangeAddressGenWithState
```